requirements-dev.txt

```diff
@@ -15,8 +13,9 @@ nbsphinx
 pydocstyle
 pytest <7.1
 sphinx_compas_theme >=0.15.18
-sphinx ==4.5
+matplotlib
+sphinx
+cryptography == 37.0.4
```

Comment:
Hard-pinning `cryptography` sounds a bit complicated in terms of dependencies, doesn't it? If this absolutely is needed, maybe add a comment somewhere explaining why and when it can be loosened.

Reply:
yep that one is super annoying, I don't like it at all too, I'll triple check if there is any way around, otherwise I'll add back the flag that reminds to remove it when it's fixed

Reply:
but why would we need to do something here that otherwise is not necessary? where does the need for pinning come from?

Reply:
if i m not mistaken, cryptography is required by pyopenssl, which is required by sphinx, latest version of it raise an error when building docs in ubuntu

Reply:
is this also true if we ignore python 3.7?

Reply:
btw guys this cryptography thing is now fixed by their new release, this sphinx nightmare is finally finished...